pkg/apis/core/validation/validation.go

```diff
@@ -3577,6 +3599,16 @@ func ValidateSeccompPodAnnotations(annotations map[string]string, fldPath *field
 	return allErrs
 }
 
+// ValidateSeccompProfileType tests that the argument is a valid SeccompProfileType.
+func ValidateSeccompProfileType(fldPath *field.Path, seccompProfileType core.SeccompProfileType) *field.Error {
+	switch seccompProfileType {
+	case core.SeccompProfileLocalhost, core.SeccompProfileTypeDefault, core.SeccompProfileUnconfined:
+		return nil
+	default:
+		return field.NotSupported(fldPath, seccompProfileType, []string{string(core.SeccompProfileUnconfined), string(core.SeccompProfileUnconfined)})
```

Comment:
this should enumerate the three valid values

Reply:
Fixed now.